docs/userguide.md

```diff
@@ -305,3 +311,12 @@ federation deployments:
 ```bash
 ./scripts/delete-federation.sh cluster2
 ```
+
+The above script will unjoin the host cluster from federation control panel automatically.
```

Comment:
Why do these two lines (315, 316) remain, when you have added the text starting from line 318 ?

Reply:
Fixed, thanks @irfanurrehman